src/framework/theme/components/alert/_alert.component.theme.scss

```diff
@@ -16,123 +16,32 @@
-
-    background: nb-theme(alert-bg);
-    color: nb-theme(alert-outline-fg);
-    // TODO: move alert margin style to layout
-    margin-bottom: nb-theme(alert-margin);
+    background-color: nb-theme(alert-background-color);
     border-radius: nb-theme(alert-border-radius);
     box-shadow: nb-theme(alert-shadow);
+    color: nb-theme(alert-text-color);
+    font-family: nb-theme(alert-text-font-family);
+    font-size: nb-theme(alert-text-font-size);
+    font-weight: nb-theme(alert-text-font-weight);
+    line-height: nb-theme(alert-text-line-height);
     padding: nb-theme(alert-padding);
 
-    @include nb-scrollbars(
-        nb-theme(scrollbar-fg),
-        nb-theme(scrollbar-bg),
-        nb-theme(scrollbar-width));
-
-    &.status {
-      color: nb-theme(alert-fg);
-    }
-
-    &.xxsmall-alert {
-      height: nb-theme(alert-height-xxsmall);
-    }
-    &.xsmall-alert {
-      height: nb-theme(alert-height-xsmall);
-    }
-    &.small-alert {
-      height: nb-theme(alert-height-small);
-    }
-    &.medium-alert {
-      height: nb-theme(alert-height-medium);
-    }
-    &.large-alert {
-      height: nb-theme(alert-height-large);
-    }
-    &.xlarge-alert {
-      height: nb-theme(alert-height-xlarge);
-    }
-    &.xxlarge-alert {
-      height: nb-theme(alert-height-xxlarge);
-    }
-
-    &.active-alert {
-      background-color: nb-theme(alert-active-bg);
-    }
-    &.disabled-alert {
-      background-color: nb-theme(alert-disabled-bg);
-      color: nb-theme(alert-disabled-fg);
-    }
-    &.primary-alert {
-      background-color: nb-theme(alert-primary-bg);
-    }
-    &.info-alert {
-      background-color: nb-theme(alert-info-bg);
-    }
-    &.success-alert {
-      background-color: nb-theme(alert-success-bg);
-    }
-    &.warning-alert {
-      background-color: nb-theme(alert-warning-bg);
-    }
-    &.danger-alert {
-      background-color: nb-theme(alert-danger-bg);
-    }
-
-    &.accent {
-      border-top-style: solid;
-      border-top-width: nb-theme(alert-border-radius);
+    // TODO: move alert margin style to layout
+    margin-bottom: nb-theme(alert-bottom-margin);
 
-      & nb-alert-header {
-        border-radius: 0;
-      }
-    }
+    @include nb-scrollbars(
+        nb-theme(alert-scrollbar-color),
+        nb-theme(alert-scrollbar-background-color),
+        nb-theme(alert-scrollbar-width));
 
-    &.accent-active {
-      border-top-color: nb-theme(alert-active-bg);
-    }
-    &.accent-disabled {
-      border-top-color: nb-theme(alert-disabled-bg);
-    }
-    &.accent-primary {
-      border-top-color: nb-theme(alert-primary-bg);
-    }
-    &.accent-success {
-      border-top-color: nb-theme(alert-success-bg);
-    }
-    &.accent-info {
-      border-top-color: nb-theme(alert-info-bg);
-    }
-    &.accent-warning {
-      border-top-color: nb-theme(alert-warning-bg);
-    }
-    &.accent-danger {
-      border-top-color: nb-theme(alert-danger-bg);
+    &.disabled {
```

Comment:
I don't think we need a `disabled` alert. It was a part of available statuses in Nebular 3, let's remove it now.
Unless you see a usecase.

Reply:
Removed in 375931a6d89223a0ecc213c58e52050a3ab20372